src/value.mjs

```diff
@@ -24,7 +24,7 @@ import { ValueMap, OutOfRange } from './helpers.mjs';
 
 // #sec-ecmascript-language-types
 export class Value {
-  constructor(value) {
+  constructor(value = undefined, length = undefined, name = undefined, prefix = undefined) {
```

Comment:
this can just leave out name/length/prefix. if someone really cares they can use `CreateBuiltinFunction` directly, as it is publicly exported.